lib/main.js

```diff
@@ -116,7 +119,11 @@ event_sources.json and ${program.eventFile} files as needed.`)
       console.log(JSON.stringify(results))
     }
 
-    handler(event, context, callback)
+    const nameSpace = createNamespace('AWSXRay')
+    nameSpace.run(() => {
+      nameSpace.set('segment', new AWSXRay.Segment('annotations'))
+      handler(event, context, callback)
+    })
```

Comment:
I'm not familiar with X-Ray too much, does this change mean that everyone needs to have it enabled?